server/src/resolvers.js

```diff
@@ -4,16 +4,147 @@ const {User} = require('./resolvers/user');
 const {Event} = require('./resolvers/event');
 
 module.exports = {
-  Query,
+  Query: {
+    events: (_, {pageSize, after}, context) => {
+      // Temporary data for test event list page.
+      const events_list = [
+        {
+          id: 0,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'Is Sang-geon Good?',
+          description: 'Learning Ethics by analysing behavior of Sang-geon Yun',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: [],
+        },
+        {
+          id: 1,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'Introduction to Algorithm',
+          description: 'Basic to ICPC World Final... You think you can handle this?',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: [],
+        },
+        {
+          id: 2,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'What is meaning of life?',
+          description: 'Sometimes we thinking about huge size of the universe. ' +
+              'Compare to what humanity accomplished so far, the universe seems ' +
+              'wast of space... isn\' it?',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: [],
+        },
+        {
+          id: 3,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'If AI gets self awareness...',
+          description: 'That means bad thing to human? YES. But in a perspective ' +
+              'of evolution? NO.',
+          price: 999999.99,
```

Comment:
Is this the max value of this field? If so, move value out and make it as a constant

Reply:
That was just temporary value